### PR TITLE
Add CLI argument to disable log color output

### PR DESCRIPTION
**Pull request by bkchr**
Fixes: https://github.com/paritytech/polkadot/issues/2153

polkadot companion: https://github.com/paritytech/polkadot/pull/2180

### CODE REVIEW

**Review by shawntabrizi — APPROVED** [review]

File: client/rpc/src/system/tests.rs [hunk 1]
```diff
@@ -344,13 +344,15 @@ fn test_add_reset_log_filter() {
...
 	if std::env::var("TEST_LOG_FILTER").is_ok() {
-		sc_cli::init_logger("test_before_add=debug", Default::default(), Default::default(), false).unwrap();
+		sc_cli::init_logger(
+			sc_cli::InitLoggerParams { pattern: "test_before_add=debug".into(), ..Default::default() },
+		).unwrap();
 		for line in std::io::stdin().lock().lines() {
 			let line = line.expect("Failed to read bytes");
 			if line.contains("add_reload") {
-				assert!(api(None).system_add_log_filter("test_after_add".to_owned()).is_ok(), "`system_add_log_filter` failed");
+				api(None).system_add_log_filter("test_after_add".into()).expect("`system_add_log_filter` failed");
```
Comment (shawntabrizi) [inline client/rpc/src/system/tests.rs:353]: DQ, why is `into()` better than `to_owned()` here?

Comment (bkchr) [reply client/rpc/src/system/tests.rs:353]: No, difference. Just less code 😂

**Review by thiolliere — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Comment by bkchr** [comment]
bot merge

**Pull request merged**